server/src/main/java/org/opentosca/toscana/plugins/kubernetes/KubernetesPlugin.java

```diff
@@ -35,67 +26,14 @@ private static Platform getPlatformDetails() {
         Set<Property> platformProperties = new HashSet<>();
         return new Platform(platformId, platformName, platformProperties);
     }
-
+    
     @Override
-    public void transform(TransformationContext context) throws Exception {
-        logger.info("Started transformation to kubernetes artifact.");
-        PluginFileAccess fileAccess = context.getPluginFileAccess();
-        // TODO rewrite: context does no longer provide TServiceTemplate, but the EffectiveModel
-//        DockerApp dockerApp = getDockerApp(context.getModel());
-//        String appName = dockerApp.getTag()[1];
-//        generateReadme(fileAccess, appName);
-//        generateResources(fileAccess, appName);
-//        copyImplementationArtifacts(fileAccess, dockerApp);
-//        generateBuildScript(fileAccess, dockerApp);
-    }
-
-    private void generateBuildScript(PluginFileAccess fileAccess, DockerApp dockerApp) throws IOException, DockerImageBuildScript.DockerImageBuildScriptException {
-        DockerImageBuildScript buildScript = new DockerImageBuildScript();
-        buildScript.addDockerApp(dockerApp);
-        fileAccess.access("/scripts/build.sh").append(buildScript.generateBuildScript()).close();
-    }
-
-    private void generateResources(PluginFileAccess fileAccess, String appName) throws IOException {
-        String resourceFile = new KubernetesResourceFileCreator().createResourceFileAsString(appName);
-        String resourceFilePath = "/" + appName + "_resource.yaml";
-        fileAccess.access(resourceFilePath).append(resourceFile).close();
-    }
-
-    private void generateReadme(PluginFileAccess fileAccess, String appName) throws IOException {
-        String manual = KubernetesManualCreator.createManual(appName, (appName + "_resource.yaml"));
-        String manualPath = "/Readme.md";
-        fileAccess.access(manualPath).append(manual).close();
+    protected Set<Class<?>> getSupportedNodeTypes() {
```

Comment:
As already discussed this method is not needed anymore (delete it from its superclass)

Reply:
Yeah started this before I submitted the other PR so in this context it is still valid ;)

Reply:
Did merge the other one about a minute ago, just update this branch :)